scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -271,16 +271,30 @@
   </div>
   <div class="row">
     <div class="col-2">
-      <a class="btn btn-secondary btn-sm" data-toggle="collapse" href="#stringcoords" aria-expanded="false" aria-controls="stringcoords">
+      <a class="btn btn-secondary btn-sm form-control" data-toggle="collapse" href="#stringcoords" aria-expanded="false" aria-controls="stringcoords">
         String coordinates
       </a>
     </div>
-    <div class="col-3 offset-1">
-      <div id="stringcoords" class="collapse">
-        {{ wtf.form_field(form.chrom_pos) }}
+    <div class="col-4">
+      <div id="stringcoords" class="collapse form-group row">
```

Comment:
Would be nice if the div stays uncollapsed after a search that contains compound rank score and/or follow filter, if it's not super complicated, otherwise it's fine!

Reply:
Right, I'll give it a shot!